src/vision.cpp

```diff
@@ -124,6 +125,19 @@ bool Vision::configure()
     node_->set_parameter(rclcpp::Parameter(FRAME_ID_PARAM, frame_id_));
   }
 
+  node_->declare_parameter<double>(MAX_PUB_RATE_PARAM);
+  if (!node_->get_parameter<double>(MAX_PUB_RATE_PARAM, max_pub_rate_hz_))
+  {
+    max_pub_rate_hz_ = 30.0;
```

Comment:
What happens if you set this to a higher publisher rate than the camera is streaming at? iirc it maxes out at 30 hz, but can be as low as 6 for depth images? Does it matter?

Reply:
Yeah it just looks like `rate.sleep()` doesn't make any difference to the frequency of publication when it's set to a higher frequency than the camera streams at